### PR TITLE
Update Pods

### LINKED ISSUE #930: Update pods
Update as many pods as possible to support Swift 5.

Update pods in the project:

- [x] Tracks - updated in prep for Swift 5
- [x] Authenticator - updated in prep for Swift 5
- [x] CocoaLumberjack - supports Swift 5
- [x] KeychainAccess - supports Swift 5

**Do not** update Charts to v. 3.3.0. Although it contains Swift 5 support, it is unstable and crashes. 3.2.2 is the current stable version.

**Known issues:** CocoaLumberjack 3.5.3 is required for the latest Authenticator changes. You will need to resolve conflicting version requirements in the project.

### PR DESCRIPTION
**Pull request by mindgraffiti**
Closes #930.

In this PR I have updated as many pods as possible, in preparation for Swift 5 support or because they already support Swift 5.

## To Test
- check out this branch
- `bundle exec pod install`
- run unit tests
- build & run

Check that there are no warnings in Terminal when installing pods. Check that the app runs as expected. Check that all unit tests pass.

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`. n/a

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -59,8 +59,8 @@ end
 #
 def yosemite_pods
   pod 'Alamofire', '~> 4.7'
-  pod 'CocoaLumberjack', '~> 3.4'
-  pod 'CocoaLumberjack/Swift', '~> 3.4'
+  pod 'CocoaLumberjack', '~> 3.5'
+  pod 'CocoaLumberjack/Swift', '~> 3.5'
```
Comment (mindgraffiti) [inline Podfile]: Are these individual definitions for our internal frameworks (Yosemite / Storage / Networking) still required?

File: Podfile [hunk 2]
```diff
@@ -135,7 +135,7 @@ end
 pre_install do |installer|
   installer.analysis_result.specifications.each do |s|
     if s.name == 'CocoaLumberjack'
-      s.swift_version = '4.2'
+      s.swift_version = '5.0'
```
Comment (mindgraffiti) [inline Podfile]: I don't know if I did this right.

Comment (ctarda) [reply Podfile]: To be honest, I don't know what this does.

**Comment by mindgraffiti** [comment]
Thank you @ctarda!

**Pull request merged**